src/main/flight/mixer.c

```diff
@@ -661,6 +661,130 @@ void StopPwmAllMotors()
 }
 
 #ifndef USE_QUAD_MIXER_ONLY
+static int16_t decidegreeToServoPulse(uint8_t servo_index, int16_t angle_deci_degree)
+{
+    // normalize angle to range -180 to 180 (degree)
+    while (angle_deci_degree > 1800)
+        angle_deci_degree -= 3600;
+    while (angle_deci_degree < -1800)
+        angle_deci_degree += 3600;
+
```

Comment:
never used a multi-turn servo so i'm not sure that would be the only catch. It could make sense to remove the normalization or move it in a different function maybe.
I don't really know, i think i would prefer some specialized function instead of a generic one.